src/validation.cpp

```diff
@@ -1054,377 +1026,355 @@ bool AcceptToMemoryPoolWorker(CTxMemPool& pool, CValidationState& state, const C
         view.SetBackend(dummy);
         } // LOCK
 
-        if (!tx.IsZerocoinSpend() && !tx.IsZerocoinRemint()) {
```

Comment:
could you please remove `if` and leave braces and thus retaining formatting?

Reply:
done